src/addons/__tests__/ReactComponentWithPureRenderMixin-test.js

```diff
@@ -50,20 +50,20 @@ describe('ReactComponentWithPureRenderMixin', function() {
       getInitialState: function() {
         return {
           cut: false,
-          slices: 1,
+          slices: 1
```

Comment:
I agree, and think we should actually enforce trailing commas, but I think that should be another commit, and could be done with a codemod.